packages/api/src/schema/types/order/order.ts

```diff
@@ -81,5 +81,13 @@ export default [
       updated: DateTime
       user: User
     }
+
+    type OrderStatistics {
```

Comment:
Ask chatgpt to make these type names consistent, it's like a mix of past, present, verbs and nouns currently with no alignment.

Maybe use orderedCount instead of checkoutCount, confirmedCount and rejectedCount or:

- newCount
- checkoutCount
- rejectCount
- confirmCount
- fullfillCount

or:

- creationCount
- orderCount
- rejectionCount
- ..

you get the idea i guess